### ISSUE TITLE
Release for Dropwizard 2.x.x

### ISSUE THREAD

**Issue by psouquet** [benjamin-bader/droptools#59]
Hi, 
when do you plan on releasing `dropwizard-jooq` for dropwizard 2.x.x ?
I'm trying to update to dw 2.0.2 and it seems your very useful lib is the last missing piece !

Related question: do you push your snapshot builds on a public maven somewhere ? (And would you be willing to do so ?) That would help me test in the meantime.

Thanks a lot for your work !

**Comment by benjamin-bader** [comment]
I haven't been a dropwizard user for many years, so my work here is limited to maintenance like pushing new releases when folks need them :)

Funny, I thought I _did_ the v2 work already.  Maybe I just forgot to release it 😨

**Comment by benjamin-bader** [comment]
I do have snapshots pubished to Sonatype's snapshot repo, so that might unblock you until I can get the new release published.

**Comment by benjamin-bader** [comment]
Just pushed version 2.0.2-0 to Maven Central; it ought to be available within the next 15 minutes.

Thanks for your patience - took a while to get this new laptop properly set up for deploying releases.  Please give the new version a try - if there's a problem, open a new issue and I'll take a look.

**Comment by psouquet** [comment]
Thanks a lot !
